packages/libraries/cli/src/commands/app/create.ts

```diff
@@ -123,7 +132,7 @@ export default class AppCreate extends Command<typeof AppCreate> {
               );
             }
           }
-          this.error(result.addDocumentsToAppDeployment.error.message);
```

Comment:
instead of using the `this.error` function, each error has an `exitCode` property that sets the exit code.